src/algorithms/audioproblems/startstopcut.cpp

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (C) 2006-2016  Music Technology Group - Universitat Pompeu Fabra
+ *
+ * This file is part of Essentia
+ *
+ * Essentia is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU Affero General Public License as published by the Free
+ * Software Foundation (FSF), either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the Affero GNU General Public License
+ * version 3 along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+
+#include "startstopcut.h"
+#include "essentiamath.h"
+
+using namespace std;
+
+namespace essentia {
+namespace standard {
+
+const char* StartStopCut::name = "StartStopCut";
+const char* StartStopCut::category = "Audio Problems";
+const char* StartStopCut::description = DOC(
+    "This algorithm outputs if there is a cut at the beginning or at the end "
+    "of the audio by measuring the first and last non-silent frames and "
+    "comparing them to the actual beginning and end of the audio. If the first "
```

Comment:
"comparing their positions..."

Reply:
What about something like this? 

"The input audio is considered to be cut at the beginning (or the end) and the corresponding flag is activated if the first (last) non-silent frame occurs before (after) the configurable time threshold."